README.md

```diff
@@ -92,6 +92,22 @@ parser.ClearColumnsAsText();
 exporter.ClearColumnsAsText();
 ```
 
+## Allow unsanitized columns
+
+The `Exporter` supports preventing specific columns from being sanitized. This is primarily to aid numberical columns with negative signs, but can be used for any column.
```

Comment:
Could we give an example of when this might be an aid?

Reply:
Done.